docs/overview.md

```diff
@@ -0,0 +1,135 @@
+a moderate to large scale, but Sprocket should work reasonably well for many
+other users too. In particular, we aim to keep Sprocket as simple as we possibly
+can—only introducing complexity when needed to achieve the project's other
+stated goals.
+
+### Modern Development Tools
+
+A suite of development tools are included alongside the execution engine and
+package manager. We believe that, when you _do_ need to dust off your code
+editor and write an analysis workflow, that activity should as enjoyable as it
+can be. Supporting tools, such as
+[linters](https://github.com/stjude-rust-labs/wdl/tree/main/wdl-lint),
+formatters, [syntax
+highlighting](https://github.com/stjude-rust-labs/sprocket-vscode/blob/main/syntaxes/wdl.tmGrammar.json),
+and overall [editor
+integration](https://github.com/stjude-rust-labs/wdl/tree/main/wdl-lsp), are
+critical to making that vision a reality.
+
+### Community-Focused Codebase
+
+Since the beginning, Sprocket has been architected to maximize the potential for
+any interested party to effectively contribute to the codebase. This means that
+(a) every part of the code is designed to be read and modified by anyone [there
+are no "owners" of any code—only "experts" in that code], (b) strict practices,
+such as required documentation for public members, a common commit message
+style, and `CHANGELOG.md` entries, are enforced for each contribution, and (c) a
+comprehensive testing suite and CI/CD integration mean you can tinker with
+confidence.
+
+### Open, Tailored Standard
+
+Predictability, openness, and ensured longevity are virtues the project attempts
+to uphold. Given that the project is ultimately limited by the upstream workflow
+language in these regards, an open standard is critical to justify such a
+significant amount of community effort. Though many such standards exist,
+generality is, in some respects, at odds with creating a tailored experience
+within bioinformatics specifically.
+
+## Goal-Adjacent
+
+The following are "goal adjacent", meaning that the project values these things
+when they do not otherwise inhibit the primary goals.
+
+* **Providing robust implementations natively for a handful of reference
+  backends.** Outside of a few core backends that are used to drive the
+  development of the project, backends are intended to be developed and
+  maintained by the backend providers themselves. When that isn't possible,
+  custom backends should be able to be configured by end-users with a moderate
+  amount of effort.
+
+### Non-goals
+
+The following are non-goals of the project.
+
+* **Supporting multiple workflow languages or standards.** Ironically, workflow
+  languages are ideally a simple means to an end: to run large-scale
+  bioinformatics analyses with as little development and operational friction as
+  possible. We stand unconvinced that there is a strong technical argument for
+  multiple workflow languages existing within bioinformatics.
```

Comment:
I think this begs the question, "if there should only be one workflow language, **why WDL?**"

Do we need a paragraph justifying our backing of WDL over Nextflow, CWL, etc?

Reply:
At least for me, when combining the pieces about who our target audience is and the need to develop based on an open, community-driven standard, the choice is clear (though it might could just be communicated succinctly this way).

Reply:
I put a sentence here about WDL. I think it would be good to add a future page regarding the details of why we chose WDL.